python/paddle/fluid/layers/nn.py

```diff
@@ -5036,6 +5037,45 @@ def gather(input, index):
     return out
 
 
+def scatter(input, index, updates):
+    """
+    **Scatter Layer**
+
+    Output is obtained by updating the input on selected indices on the first
+    axis.
+
+    .. math::
+
+        Out = X
+        Out[Ids] = Updates
+
+    Args:
+        input: The source input with rank>=1.
```

Comment:
input (Variable): xxx

the same as below.

Reply:
Done